index.rst

```diff
@@ -302,107 +305,152 @@ Is this case, we could have something like:
+#.  An additional configuration file provides overrides for the configuration parameters set by the previous files.
 
-    - See this :download:`example _base.yaml <_static/_base.yaml>` for the ATSpectrograph CSC.
+    - See this :download:`configuration parameter override example file <_static/ATSpectrograph_example_config.yaml>` for the ATSpectrograph CSC.
+    - This file is the third configuration file to get loaded by the CSC and can override any previously declared values.
+    - These files are loaded using the `configurationToApply` attribute in the ``start`` command
+    - These are not expected to be required as part of regular operations and are meant to be used when a non-standard configuration is required
+    - If an override configuration file is also site specific, then a prefix should be added indicating which site it belongs with (e.g. ``summit_reduced_stage_travel.yaml``)
 
 #.  The labels file (e.g. ``_labels.yaml``) shall continue to exist with the same format and purpose.
-#.  Additional configuration files can provide new values for individual configuration parameters.
 
-    - See this :download:`example configuration file <_static/ATSpectrograph_example_config.yaml>` for the ATSpectrograph CSC.
+        - No label shall exist having the name ``default``
+
+#.  If a CSC receives a ``start`` command with an empty ``configuration`` (see :ref:`renaming proposal <section-renaming>`) attribute, it shall load the values in ``_init.yaml`` then the site-specific file (e.g. ``_summit.yaml``).
+#.  If a CSC receives a ``start`` command with a ``configuration`` attribute equal to a label in ``_labels.yaml``, it shall load the values in ``_init.yaml``, then then the site-specific file (e.g. ``_summit.yaml``), and lastly the override file.
 
-#.  If a CSC receives a ``start`` command with an empty ``configuration`` (see :ref:`renaming proposal <section-renaming>`) attribute, it shall load the values in ``_base.yaml``.
-#.  If a CSC receives a ``start`` command with a ``configuration`` attribute equal to a label in ``_labels.yaml``, it shall load the values in ``_base.yaml`` first and override those values defined in the mapped configuration file.
-#.  The name ``default`` shall not be used for labels.
 #.  All valid configurations shall have a label to be loaded by the CSC.
-    Configurations cannot be loaded by filename.
-#.  The configuration repository should be kept clean of configurations used for unit testing or for different purposes (e.g. test stand configurations).
-    Configurations tailored for test stand can be kept in different branches/tags.
+
+#.  The configuration repository shall not contain configurations used for unit testing.
+    Override configurations that are site specific should contain the site name as a prefix to the filename (e.g. ``summit_simple_algorithm.yaml``).
```

Comment:
I think you inserted this in the wrong places. Line 433 (just below) looks like part of line 431 (just above "Override configurations that are site specific...")